shared-module/displayio/display_core.c

```diff
@@ -283,6 +294,14 @@ void displayio_display_core_set_region_to_update(displayio_display_core_t* self,
         data[data_length++] = y2 >> 8;
         data[data_length++] = y2 & 0xff;
     }
+    // Quirk for SH1107 "column_and_page_addressing"
+    //      Page address command = 0xB0
+    if (column_and_page_addressing) {
+        data[0] = 0xB0 | (y1 & 0x07);
+        data_length = 1;
+    }
+    self->send(self->bus, data_type, chip_select, data, data_length);
+
     self->send(self->bus, data_type, chip_select, data, data_length);
```

Comment:
I haven't read the datasheet or anything, but this repeated send of the same data looks fishy.  Is it intended?

Reply:
There's only supposed to be one send.  I'll check my code again.